config/checkstyle-checks.xml

```diff
@@ -46,6 +46,26 @@
     <property name="files"
               value=".*[\\/]src[\\/](test|it|xdocs-examples)[\\/].*(?&lt;!Support)\.java"/>
   </module>
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="MultipleStringLiterals"/>
+    <property name="files" value=".*[\\/]Checker\.java"/>
+  </module>
```

Comment:
No suppressions, we need to reuse same constant.

Reply:
suppression removed